optics_functions/utils.py

```diff
@@ -115,28 +116,46 @@ def split_complex_columns(df: pd.DataFrame, columns: Sequence[str],
     return df
 
 
-def switch_signs_for_beam4(df_twiss: pd.DataFrame,
```

Comment:
To do it also for beam 4 is very ambitious. Is there also a test for this part?

Reply:
There is a test for this function, that after running a beam4 twiss through it, out comes something that looks like a beam2 twiss. this function is used for the RDTs. Beam 4 has not been tested together with coupling.